src/osd/OSD.cc

```diff
@@ -6124,9 +6124,9 @@ void OSD::do_command(Connection *con, ceph_tid_t tid, vector<string>& cmd, buffe
       f->dump_int("blocksize", bsize);
       f->dump_unsigned("bytes_per_sec", rate);
       f->close_section();
-      f->flush(ss);
+      f->flush(ds);
     } else {
-      ss << "bench: wrote " << byte_u_t(count)
+      ds << "bench: wrote " << byte_u_t(count)
```

Comment:
if ds is stdout, writing debug information to stdout is wrong. JSON MUST go to stdout, debug MUST go to stderr